build.gradle

```diff
@@ -48,7 +37,7 @@ subprojects {
 
     ext.guavaVer  = '26.0-jre'
```

Comment:
_Oooo, Guava!_ :scream: 

Out of interest, can you identify what we're using this for please? (Would it be terrible if it were to "disappear"?)

Reply:
I'm all for making things dissapear, I don't think it is used at all.

Reply:
:sparkles: :mage_man: :sparkles: 
It would be "Really Nice" :tm: if Quasar lost its dependency on Guava...

Reply:
Having checked it is used in quite a lot of the core classes. Would be a bit of a mission to take it out.

Reply:
:cry: